scripts/bootstrap

```diff
@@ -8,3 +8,27 @@ docker-compose run --entrypoint=npm client install
 docker-compose run --entrypoint=bower client install --allow-root
 # Initialize database
 docker-compose run server rake db:setup
+# Boot app
+docker-compose start
+
+# Get docker-machine ip if there is one
+port="$(docker-compose port nginx 80 | cut -d: -f2)"
+[ -z "$DOCKER_MACHINE_NAME" ] && host='localhost' || host="$(docker-machine ip)"
```

Comment:
So I should use `docker-machine ip $DOCKER_MACHINE_NAME`? I thought it would return the currently-env'd machine, because I have 5 machines and it worked :u